binderbot/binderbot.py

```diff
@@ -90,215 +70,21 @@ async def start_binder(self, timeout=3000, spawn_refresh_time=20):
                 self.log.msg(f'Binder: Waiting on event stream (phase: {phase})', action='binder-start', phase='event-stream')
```

Comment:
Suggest checking that the `notebook_url` and `token` variables are set before proceeding to the next piece of logic. This will help a little bit with debugging #35.

```suggestion
                self.log.msg(f'Binder: Waiting on event stream (phase: {phase})', action='binder-start', phase='event-stream')
                
        try:
            self.notebook_url
            self.token
        except AttributeError as e:
            raise OperationError from e
```